SpringProject/src/it/java/com/gradproject2019/user/UserControllerIT.java

```diff
@@ -176,6 +180,36 @@ public void shouldReturn200WhenGettingEmployeeById() throws URISyntaxException {
         Assert.assertEquals(employee.getOccupation(), response.getBody().getOccupation());
     }
 
+    @Test
+    public void shouldReturn200WhenGettingEmployeesByUserId() {
+        ResponseEntity<List<UserResponseDto>> response = getUserByManagerId(uri3);
+
+        Assert.assertEquals(200, response.getStatusCodeValue());
+        Assert.assertEquals(response.getBody().size(), 1);
+        Assert.assertEquals(savedUserWithManager.getId(), response.getBody().get(0).getId());
+        Assert.assertEquals(savedUserWithManager.getFirstName(), response.getBody().get(0).getFirstName());
+        Assert.assertEquals(savedUserWithManager.getUsername(), response.getBody().get(0).getUsername());
+        Assert.assertEquals(savedUserWithManager.getLastName(), response.getBody().get(0).getLastName());
+        Assert.assertEquals(savedUserWithManager.getEmail(), response.getBody().get(0).getEmail());
+        Assert.assertEquals(savedUserWithManager.getOccupation(), response.getBody().get(0).getOccupation());
+    }
+
+    @Test
+    public void shouldReturn401WhenUserUnauthorisedToGetEmployees() {
+        ResponseEntity<ErrorEntity> response = getUserByManagerIdAuthError(uri3);
+
+        Assert.assertEquals(401, response.getStatusCodeValue());
+    }
+
+    @Test
+    public void shouldReturnEmptyListWhenUserHasNoEmployees() {
+
+        ResponseEntity<List<UserResponseDto>> response = getUserByManagerIdNoneManager(uri4);
+
+        Assert.assertEquals(200, response.getStatusCodeValue());
+        Assert.assertEquals(0, response.getBody().size());
+    }
+
     @Test
     public void shouldReturn401WhenGettingUserByIdUnauthorisedDueToToken() {
```

Comment:
I think a bad token is the only way to be unauthorized, so 'due to token' is redundant here.